bwresources.py

```diff
@@ -206,7 +206,7 @@ def num_mentions(self, **kwargs):
 
     def get_topics(self, **kwargs):
         params = self._fill_mentions_params(kwargs)
-        return self.project.get(endpoint="data/volume/topics/queries", params=params)
+        return self.project.get(endpoint="data/volume/topics/queries", params=params)["topics"]
```

Comment:
The API result returns a JSON of the form `{"topics": stuff}`.  It seems convenient to just return the stuff.